### DISCUSSION TITLE
Connect SQLDB on localhost to BudiBase via XAMMP

### DISCUSSION THREAD

**Discussion by minomi89** [Budibase/budibase#9118] [Help]
![image](https://user-images.githubusercontent.com/84246956/208536784-46547f4d-dc76-4899-b721-fc25fbc7bab0.png)
![image](https://user-images.githubusercontent.com/84246956/208536820-1f12cfc4-0fd9-4db6-adaf-fa29e4f7f04e.png)
![image](https://user-images.githubusercontent.com/84246956/208536848-e0e2796d-b112-4c67-819b-aa9cfcced166.png)
![image](https://user-images.githubusercontent.com/84246956/208536916-790ad642-a628-41c6-8e0f-d6ffb958e4a9.png)

Can you help me please?

**Hosting**

- Self
  - Method: <method> 
  - Budibase Version: <version> 
  - App Version: <version> 
- Cloud
  - Tenant ID: <tenantId> 

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**
  
**App Export**
If possible - please attach an export of your budibase application for debugging/reproduction purposes.

**Desktop (please complete the following information):**

**Additional context**

**Comment by shogunpurple** [comment]
@minomi89 - you are running on the cloud budibase and trying to connect to a MySQL database running on your local machine. `localhost` will not work as they are not on the same local network - you need to use a cloud hosted DB and connect to it using the IP address of the database. Converting to a discussion as it's not a bug

**Comment by minomi89** [comment]
i'm sorry,
i need to connect a DB to try BudiBase with SQL.
I have read all the documentation but i can't connect from xammp or Google Cloud.

There is a guide to do it?
